egs/librispeech/ASR/pruned_transducer_stateless7/export-onnx.py

```diff
@@ -159,6 +160,26 @@ def add_meta_data(filename: str, meta_data: Dict[str, str]):
     onnx.save(model, filename)
 
 
+def num_tokens(
```

Comment:
Could you move this function to `icefall/utils.py` so that it is shared by all `export.py`? 

It can reduce code duplication.

Reply:
OK!